README.md

```diff
@@ -8,6 +8,38 @@ First, obtain an App ID and an API key from https://docs.traveltime.com/api/over
 
 You will need to email sales@traveltime.com or book a demo at https://traveltime.com/book-demo to gain access to TimeFilterFast(Proto).
 
+### Testing Each Endpoint Locally
+
+Install [K6](https://k6.io/docs/get-started/installation/)
+
+#### time-map
+
+```bash
```

Comment:
add also instructions how to run it with docker

Reply:
Made a general running with docker instruction which fits all endpoints. Let me know if you want a separate one for each endpoint (imo there's no need, all env variables are explained in the "running locally" part of readme)